Doc/conf.py

```diff
@@ -237,5 +237,3 @@
 # bpo-40204: Disable warnings on Sphinx 2 syntax of the C domain since the
 # documentation is built with -W (warnings treated as errors).
 c_warn_on_allowed_pre_v3 = False
```

Comment:
Also part of this "Sphinx 2 and Sphinx 3 compatibility" section:

Are `c_allow_pre_v3` and `c_warn_on_allowed_pre_v3` above covered by https://github.com/python/cpython/issues/93738? Or can they removed here?

Reply:
If you remove it you get around 1,300 lines of warnings and then a build failure... Hence #93738! Yeah, they need to be kept here.

A